scripts/upload.sh

```diff
@@ -39,28 +52,13 @@ while [[ $# -gt 0 ]]; do
     ;;
     # Error on unrecognized parameters
     *)
-      >&2 echo "ERROR: Unrecognized parameter: $1"
       show_usage
-      exit 65
+      fatal "Unrecognized parameter: $1" 65
```

Comment:
"65" has been added in transifex.sh. Is it colliding with this old "65"?

Reply:
The exit codes are only unique within a single script, and these scripts don't call each other, so it shouldn't matter (if one script calls another and exits with the exit code from that script with `|| exit $?`, it can be a bit confusing). And this error should only happen when running this locally (instead of in a CI workflow), so you know which script you just called.

The exit codes don't *need* to be unique anyway, I just prefer using unique codes for each error instead of just exiting with a generic exit code of 1 (which can also come from a command that you're calling in the script).

Especially iOS builds often *only* show the script name and the exit code (no output / error messages), so having (at least mostly) unique exit codes is nice.